CONTRIBUTING.md

```diff
@@ -366,3 +366,79 @@ duration = float_or_none(video.get('durationMs'), scale=1000)
 view_count = int_or_none(video.get('views'))
 ```
 
+### Inline values
+
+ Always inline values.
+
+#### Example
+
+A good rule of thumb is not to create variables that will only be used once.
```

Comment:
>only be used once

This should be in definition not in example.

Reply:
Fixed.